frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -216,6 +221,49 @@ export default function CellGuideCard({
     setSelectedGene(undefined);
   }, [selectedOrgan, selectedOrganism, setSelectedGene]);
 
+  const { computationalMarkerGeneTableData } =
+    useComputationalMarkerGenesTableRowsAndFilters({
+      cellTypeId,
+      organismName: selectedOrganism.name,
+      organId: selectedOrganId,
+    });
+
+  const sdsGenesList = useMemo(() => {
+    const genes = computationalMarkerGeneTableData.map((gene) => ({
+      name: gene.symbol,
+    }));
+
+    // Insert an element at the beginning
+    genes.unshift({ name: "No Gene" });
```

Comment:
maybe extract "No Gene" as a constant, so we have single source of truth for reference?